src/@next/IndexTable/IndexTable.tsx

```diff
@@ -37,8 +46,20 @@ const IndexTable = ({
       <PolarisIndexTable
         bulkActions={bulkActions}
         checkbox={renderCheckboxHeader}
-        itemCount={itemCount}
+        itemCount={loading ? 0 : itemCount}
         selectedItemsCount={selectedItemsCount}
+        emptySearchTitle={emptySearchTitle ?? ''}
+        loading={false}
```

Comment:
I guess can remove this prop

Reply:
Sorry I forgot to mention why I added this, since I implement the loading state myself, I need to make sure that the stock `loading` state won't appear. That's why I added this prop.